WeatherXM/docs
# FE-649 FE-752 Feature/upgrade docusaurus libs

Upgraded Docusaurus and some libraries we use. Also made some changes due to build issues and some HTML fixes as the layout was changed a bit and we needed the UI to match the previous one.

## How to test?
1. Run `yarn` in the docs folder before testing it as the latest versions of the libraries we use should be fetched.
2. Ensure that **ALL** pages match the previous UI (compare local build with https://docs.weatherxm.com)

Fixes #56
